src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.body {
+  color: blue;
+}
+
+/* .hello {
+  text-align: center;
+} */
```

Comment:
remove this, it's not requited in the task and the tests can't pass because of these styles